### DISCUSSION TITLE
Change IP from Local to other Server

### DISCUSSION THREAD

**Discussion by k8jisoo** [requarks/wiki#6747] [Help / Questions]
I installed wikijs Windows based on "https://docs.requarks.io/install/windows)".
Can I change server IP?

**Comment by NGPixel** [comment]
??? What server IP?

You can change the network interface IP it binds to in your `config.yml`, which by default is `0.0.0.0`

**Reply by k8jisoo** [reply]
Now I'm using localhost:3000, but I wanna change IP to xxx.xxx.xxx.xxx:5000.
How can I chang IP and Port?

**Reply by NGPixel** [reply]
The IP has nothing to do with Wiki.js. This is set by your server.

The port can be changed in your config.yml, or just change the port mapping if running via docker.
